source/deploy/try-mattermost.rst

```diff
@@ -0,0 +1,80 @@
+Try Mattermost
+===============
+
+This guide provides instructions for quickly trying out Mattermost using either Docker or Azure Marketplace. These options are ideal for testing and evaluation purposes as they allow you to quickly get a Mattermost instance up and running
+for exploration and testing.
+
+Deployment Options
+------------------
+
+.. tab:: Azure Marketplace
```

Comment:
Should we put Docker first since it's probably the most accessible way to quick start for an evaluation? 

Following up on my comment in Hub, is there any specific reason the docker preview image is TE or can we make it E0?

Reply:
Adding @spirosoik on both asks, good questions.

Reply:
Docker Preview is currently added, the question I think is what should be the default tab. Currently it's Azure AIO VM.